buildtools/emojis.php

```diff
@@ -29,6 +29,11 @@
         if (preg_match("/^\d+/", $name)) {
             $name = "_" . $name;
         }
+	$name = str_replace("-", "minus", $name);
+	$name = str_replace("+", "plus", $name);
+	if ($name == "new") {
+		$name = "_new";
+	}
         $header .= "	constexpr const char " .$name . "[] = \"$code\";\n";
```

Comment:
Please add this too: https://github.com/brainboxdotcc/DPP/pull/822#issuecomment-1698943596

Reply:
Yes try including it in src/unittest/coro.cpp as well, it'll break.